dp/cloud/python/magma/configuration_controller/response_processor/response_db_processor.py

```diff
@@ -72,17 +73,14 @@ def process_response(self, db_requests: List[DBRequest], response: requests.Resp
             return
 
     def _populate_grant_states_map(self, session):
-        self.grant_states_map = {
-            GrantStates.GRANTED.value: session.query(DBGrantState).filter(
-                DBGrantState.name == GrantStates.GRANTED.value,
-            ).scalar(),
-            GrantStates.AUTHORIZED.value: session.query(DBGrantState).filter(
+        grant_states = session.query(DBGrantState).filter(
+            or_(
                 DBGrantState.name == GrantStates.AUTHORIZED.value,
-            ).scalar(),
-            GrantStates.UNSYNC.value: session.query(DBGrantState).filter(
+                DBGrantState.name == GrantStates.GRANTED.value,
                 DBGrantState.name == GrantStates.UNSYNC.value,
-            ).scalar(),
-        }
+            ),
+        ).all()
+        self.grant_states_map = {gs.name: gs for gs in grant_states}
```

Comment:
maybe we can also do that for other enum tables

Reply:
you mean grant and request states? they are only populated once during db initialization so don't think it makes much difference

Reply:
did that for grant_states and request_types in radio controller